Common/VR/VRFramebuffer.cpp

```diff
@@ -76,7 +76,7 @@ bool ovrFramebuffer_Create(XrSession session, ovrFramebuffer* frameBuffer, int w
 	frameBuffer->DepthSwapChain.Height = swapChainCreateInfo.height;
 
 	// Create the color swapchain.
-	swapChainCreateInfo.format = GL_RGBA8;
+	swapChainCreateInfo.format = GL_SRGB8_ALPHA8;
```

Comment:
Hm, won't this change the graphics for Quest too? And PSP graphics are not really SRGB-aware, we don't actually want color conversion, I think...

Reply:
It still looks the same on Quest. The problem is you cannot use `GL_RGBA8` on Pico because of a bug on the device (it makes for some OpenXR projection completely different colors).

Reply:
That's curious, I would have thought it would have changed things. Though GL is its own universe of brokenness I guess...